<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pictures</class>
 <widget class="QDialog" name="Pictures">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pictures</string>
  </property>
  <widget class="QPushButton" name="prevButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">&amp;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">&amp;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
